library/std/src/sys/thread_local/destructors/linux_like.rs

```diff
@@ -12,9 +12,6 @@
 
 use crate::mem::transmute;
 
-// FIXME: The Rust compiler currently omits weakly function definitions (i.e.,
-// __cxa_thread_atexit_impl) and its metadata from LLVM IR.
-#[no_sanitize(cfi, kcfi)]
```

Comment:
@rcvalle, I saw you added this, do you know how to test this? I ran all standard library and compiler tests.

Reply:
The test you added in #138349 should cover this already so it should be fine.